docs/advanced/deposit.md

```diff
@@ -121,7 +121,7 @@ the requests to different contracts (facets) that can be independently updated a
 ![Diamond proxy layout](https://user-images.githubusercontent.com/128217157/229521292-1532a59b-665c-4cc4-8342-d25ad45a8fcd.png)
 
 You can find more detailed description in
-[Contract docs](https://github.com/matter-labs/zksync-2-contracts/blob/main/docs/Overview.md)
+[Contract docs](https://era.zksync.io/docs/reference/architecture/contracts/system-contracts.html)
```

Comment:
I think the correct link would be `https://github.com/matter-labs/era-contracts/blob/main/docs/Overview.md`.

Reply:
Hi @popzxc,

I've addressed the feedback and created a new pull request based on the latest code from the main branch. I've also updated the link as you recommended.

[New pull request #335 ](https://github.com/matter-labs/zksync-era/pull/335)

I've kept this original PR open for reference, but feel free to close it if the new one looks good.

Thank you for your guidance and patience for a small silly fix 💯